### PR TITLE
docs: add docs for 19.x

### PR DESCRIPTION
**Pull request by wolfy1339**

### CODE REVIEW

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -100,6 +100,11 @@ module.exports = {
             branch: "17.x", // older versions specify a branch name for this repo
             endpoints: "3.x", // ...and one for the endpoint methods
           },
+          {
+            name: "v18", // the path of the older version
+            branch: "18.x", // older versions specify a branch name for this repo
+            endpoints: "4.x", // ...and one for the endpoint methods
```
Comment (wolfy1339) [inline docs/gatsby-config.js:106]: I am not sure on this value, as gatsby gives an error from git
```
fatal: ambiguous argument 'origin/4.x': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Comment (gr2m) [reply docs/gatsby-config.js:106]: Hm the branch exists:
https://github.com/octokit/plugin-rest-endpoint-methods.js/tree/4.x

I think this should be 5.x though? Looks like we did the upgrade as part of v18.12.0

Can you check if that helps? If not I'd have to investigate myself, I'm not sure what causes the error
Is `4.x` the right one for v18?

Comment (wolfy1339) [reply docs/gatsby-config.js]: > I think this should be 5.x though? Looks like we did the upgrade as part of v18.12.0

Indeed, the correct version is `5.x` for v18

> Can you check if that helps? If not I'd have to investigate myself, I'm not sure what causes the error

The same error happens

Comment (wolfy1339) [reply docs/gatsby-config.js]: Clearing the build cache folder, and the error is resolved. It must've done a bad clone and didn't try to recover

**Review by gr2m — APPROVED** [review]
THANK YOU 🙏🏼 🙇🏼

**Pull request merged**